.mergify.yml

```
@@ -200,6 +200,31 @@ pull_request_rules:
               - "#changes-requested-reviews-by=0"
               - "approved-reviews-by=@ceph/ceph-csi-contributors"
               - "approved-reviews-by=@ceph/ceph-csi-maintainers"
+              - "status-success=DCO"
+              - "status-success=codespell"
+              - "status-success=go-test"
+              - "status-success=golangci-lint"
+              - "status-success=lint-extras"
+              - "status-success=mod-check"
```

```suggestion
              - "status-success=mod-check"
              - "status-success=commitlint"
```

commitlint is missing.

---

good catch! It seems not everything is part of the `queue_rules` for some reason :shrug: